lib/tests/suites/argonaut_sprint_6_test.rb

```diff
@@ -203,6 +207,10 @@ def validate_observation_reply(reply)
         warning { assert valid_observation_count > 0, "No laboratory Observations were found for this patient" }
```

Comment:
I know we had talked about `skip`-ing if there were no valid observations, rather than `warn`-ing; is this something we want to do?